package.json

```diff
@@ -97,6 +97,7 @@
     "start:server": "live-server data/content --port=1963 --no-browser",
     "firefox": "jpm run -b beta",
     "test": "npm-run-all test:*",
+    "pretest": "npm run bundle",
```

Comment:
Turns out, tests were failing if you ran `npm run clean` and then `npm test` because required assets weren't generated. So I added a `pre` hook here to make sure `bundle` gets called. Perhaps this was too much and slows down builds/tests though.